telefunc/tsconfig.cjs.json

```diff
@@ -4,7 +4,5 @@
     "outDir": "dist/cjs/",
     "target": "ES2017",
     "module": "CommonJS"
-  },
-  "exclude": ["./node_modules/", "./dist/", "**/*.spec.ts", "./client/"],
-  "include": ["./**/*"]
+  }
```

Comment:
This was needed for making the test suite go green. (Next.js's tests were failing because the client-side code wasn't transpiled to CJS.)

Reply:
haha, that's too precise.